Nexd/Nexd/Resources/de.lproj/Localizable.strings

```diff
@@ -36,6 +36,7 @@
 "error_message_unknown" = "Unbekannter Fehler";
 "error_message_user_detail_field_missing" = "Bitte ausfüllen";
 "error_title" = "Fehler";
+"error_message_user_terms_of_use_consent_missing" = "Bitte den Nutzungsbedingungen zustimmen";
```

Comment:
we shouldn't add strings directly to the `Localizable.strings` files, as we are using POeditor (https://poeditor.com/projects/view?id=328861) to maintain all the strings and translations used by the app. The goal is to be able to share the strings with the android app.

So every now and then we re-generate the Localizable.strings from poeditor with the latest translations which will then overwrite all manual changes to Localizable.strings.